apps/remix-ide/src/blockchain/blockchain.js

```diff
@@ -182,9 +182,8 @@ export class Blockchain extends Plugin {
         _paq.push(['trackEvent', 'blockchain', 'Deploy With Proxy', 'Proxy deployment failed: ' + error])
         return this.call('terminal', 'logHtml', log)
       }
-      if (networkInfo.name === 'VM') this.config.set('vm/proxy', address)
-      else this.config.set(`${networkInfo.name}/${networkInfo.currentFork}/${networkInfo.id}/proxy`, address)
       await this.saveDeployedContractStorageLayout(implementationContractObject, address, networkInfo)
+      this.events.emit('newProxyDeployment', address, new Date().toISOString(), implementationContractObject.contractName)
```

Comment:
you already have a `new Date().toISOString()` (see line 256), could these be merged?

Reply:
It cannot be merged because i am using this event primarily for jsvm that doesn't save past deployments.